BENCHMARKING.md

```diff
@@ -293,8 +293,10 @@ The supported environment variables are described in the following list:
     library formats (cuSPARSE with `cusp_` prefix or hipSPARSE with `hipsp_`
     prefix) can be used as well. Multiple options can be passed. The default is
     `csr,coo,ell,hybrid,sellp`.
-* `SOLVERS={bicgstab,bicg,cg,cgs,fcg,gmres}` - the solvers which should be
-    benchmarked. Multiple options can be passed. The default is `cg`.
+* `SOLVERS={bicgstab,bicg,cg,cgs,fcg,gmres,lower_trs,upper_trs}` - the solvers
+    which should be benchmarked. Multiple options can be passed.
+    Note that `lower/upper_trs` by default don't use a preconditioner, as they
+    are exact direct solvers. The default is `cg`.
```

Comment:
I found this a little bit confusing. I guess we're talking about cusparse tri-solves etc. when we say "exact direct solvers", but then it looks like the default is cg.

Maybe we could consider the following rewording?
```
* `SOLVERS={bicgstab,bicg,cg,cgs,fcg,gmres,lower_trs,upper_trs}` - the solvers
    which should be benchmarked. Multiple options can be passed. The default is `cg`.
    Note that `lower/upper_trs` by default don't use a preconditioner, as they
    are by default exact direct solvers.
```

Reply:
Great idea! BTW, you can suggest changes directly by dragging along multiple lines, adding a comment and clicking the leftmost button "Insert a suggestion"

Reply:
Ah yes, thanks for the hint. I was wondering how to do that.